packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.12.2]
+
+### Changed
+
+- Bump `@metamask/profile-sync-controller` from `^0.9.7` to `^0.9.8` ([#4902](https://github.com/MetaMask/core/pull/4902))
```

Comment:
It looks like we are bumping the peer dependency on `keyring-controller` in this release. Should this be a minor bump instead of patch and then should we mention the bump in the changelog? We are also bumping `controller-utils` to `^11.5.0` so should we include that too?

``` suggestion
## [0.13.0]

### Changed

- **BREAKING:** Bump `@metamask/keyring-controller` from `^17.0.0` to `^18.0.0` ([#4195](https://github.com/MetaMask/core/pull/4195))
- Bump `@metamask/profile-sync-controller` from `^0.9.7` to `^0.9.8` ([#4902](https://github.com/MetaMask/core/pull/4902))
- Bump `@metamask/controller-utils` from `^11.4.2` to `^11.5.0` ([#4195](https://github.com/MetaMask/core/pull/4195))